projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -915,10 +902,6 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
               field.options = this.setEvidence;
               field.range = null;
             }
-            else if (field.code === 'ecm') {
```

Comment:
is this not required?

Reply:
ecm is only required for root level
for section level it is not required